src/nom.rs

```diff
@@ -602,9 +602,10 @@ pub fn float(input: &[u8]) -> IResult<&[u8],f32> {
     recognize!(
       tuple!(
         opt!(alt!(tag!("+") | tag!("-"))),
-        alt!(
-          delimited!(digit, tag!("."), opt!(digit))
-          | delimited!(opt!(digit), tag!("."), digit)
+        alt_complete!(
```

Comment:
Probably something with eof. Your parser wants more digits until eof or a non-digit is found